app/assets/stylesheets/govuk_publishing_components/components/_title.scss

```diff
@@ -19,15 +19,15 @@
   &:link,
   &:visited {
     color: inherit;
+  }
 
-    &:focus {
-      color: $govuk-focus-text-colour;
-    }
+  &:hover {
+    text-decoration: underline;
   }
 
-  &:hover,
   &:focus {
-    text-decoration: underline;
+    text-decoration: none;
+    color: govuk-colour("black");
```

Comment:
we should use the `$govuk-focus-text-colour` for consistency

Reply:
Thanks, have updated.